### DISCUSSION TITLE
blog/15-%E5%85%AD%E6%9C%88%E7%AC%AC%E4%B8%89%E5%91%A8%E5%B0%8F%E8%AE%B0/

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#29] [Announcements]
# blog/15-%E5%85%AD%E6%9C%88%E7%AC%AC%E4%B8%89%E5%91%A8%E5%B0%8F%E8%AE%B0/

进入梅雨季的南京格外的闷热，阴雨绵绵的天气造成的体感很不舒适，浑身都觉得湿漉漉。

https://blog.writely.xyz/blog/15-%E5%85%AD%E6%9C%88%E7%AC%AC%E4%B8%89%E5%91%A8%E5%B0%8F%E8%AE%B0/

**Comment by flashlab** [comment]
我们这周边只要是环境好点的烤肉店都很火爆
